README.md

```diff
@@ -96,6 +96,19 @@ export NEAR_ENV=mainnet
 
 ---
 
+### API keys
```

Comment:
I want to mention that having "API keys" and "Access keys" might be confusing (in the rendered markdown file it is hard to notice that they have different header levels.

I would consider renaming "API keys" to "RPC server selection"

Reply:
Ok, good point. Renamed to `Custom RPC server selection`, since we still have a `standard` one.